scopes/ui-foundation/ui/webpack/webpack.base.config.ts

```diff
@@ -155,7 +155,7 @@ export default function createWebpackConfig(
             // smaller than specified limit in bytes as data URLs to avoid requests.
             // A missing `test` is equivalent to a match.
             {
-              test: [/\.bmp$/, /\.gif$/, /\.jpe?g$/, /\.png$/],
+              test: [/\.bmp$/, /\.gif$/, /\.jpe?g$/, /\.png$/, /\.svg$/],
```

Comment:
what about the svgr/webpack as you did for react?

Reply:
`svgr` converts svg to react components. I could add it to the main ui, but I don't think we need it.